assembly-syntax/src/ast/instruction/print.rs

```diff
@@ -218,12 +218,13 @@ impl PrettyPrint for Instruction {
             Self::PushFelt(value) => {
                 inst_with_felt_imm("push", &Immediate::Value(Span::unknown(*value)))
             },
-            Self::PushWord(value) => flatten(const_text("push") + const_text(".") + value.render()),
-            Self::PushU8List(values) => inst_with_pretty_params("push", values),
-            Self::PushU16List(values) => inst_with_pretty_params("push", values),
-            Self::PushU32List(values) => inst_with_pretty_params("push", values),
+            Self::PushWord(value) => flatten(const_text("push.") + value.render()),
+            Self::PushWordSlice(value, range) => flatten(
+                const_text("push.")
+                    + value.render()
+                    + text(format!("[{}..{}]", range.start, range.end)),
```

Comment:
When formatting using the `PrettyPrint` trait, it's generally a good idea to emit delimiter tokens separately, so that the formatter can choose to break lines at those delimiters.

```suggestion
                    + const_text("[")
                    + display(range.start)
                    + const_text("..")
                    + display(range.end)
                    + const_text("]"),
```